### PR TITLE
Add some tests for long tuples

### PR DESCRIPTION
**Pull request by agocke**
/cc @jcouv @VSadov @dotnet/roslyn-compiler Please review

### CODE REVIEW

File: src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Tuples.cs [hunk 1]
```diff
@@ -460,6 +470,26 @@ private void ValidateAttributesOnFields()
                 var field7 = _derivedClass.GetMember<FieldSymbol>("Field7");
                 ValidateTupleNameAttribute(field7, expectedTupleNamesAttribute: false);
                 Assert.False(field7.Type.IsTupleType);
+
+                // public static (int e1, int e2, int e3, int e4, int e5, int e6, int e7, int e8, int e9) Field8;
+                var field8 = _derivedClass.GetMember<FieldSymbol>("Field8");
+                expectedElementNames = new[]
+                {
+                    "e1", "e2", "e3", "e4", "e5",
+                    "e6", "e7", "e8", "e9", null, null
```
Comment (cston) [inline src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Tuples.cs]: What do the trailing `null`s represent?

Comment (jcouv) [inline src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Tuples.cs]: No names on the nested `ValueTuple'2`

Comment (agocke) [inline src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Tuples.cs]: Correct.

**Comment by jcouv** [comment]
LGTM

**Comment by cston** [comment]
LGTM

**Pull request merged**